src/diffusers/models/attention.py

```diff
@@ -356,11 +387,15 @@ def forward(self, hidden_states):
                     dtype=query_proj.dtype,
                     device=query_proj.device,
                 ),
-                query_proj,
-                key_proj.transpose(-1, -2),
+                query_proj_input,
+                key_proj_input,
                 beta=0,
-                alpha=scale,
+                alpha=alpha,
             )
+            if mask is not None:
+                mask = mask.repeat_interleave(self.num_heads, dim=0)
+                mask = mask.unsqueeze(1)
+                attention_scores = attention_scores + mask
```

Comment:
Ah they are using a mask? Ok good to introduce it here! Let's make sure to call it maybe `attention_mask` right away :-) 
```suggestion
                attention_scores = attention_scores + attention_mask
```

Reply:
done!